cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/KafkaMetricAnomalyFinderTest.java

```diff
@@ -51,7 +51,7 @@ public void testMetricAnomaliesWithNullArguments() {
   public void testMetricAnomalies() {
     MetricAnomalyFinder<BrokerEntity> anomalyFinder = createKafkaMetricAnomalyFinder();
     Map<BrokerEntity, ValuesAndExtrapolations> history =
-        createHistory(Collections.singletonMap(METRIC_ID, 20.0), 20, BROKER_ENTITIES.get(0));
+        createHistory(Collections.singletonMap(METRIC_ID, 20.0), Collections.singletonMap(METRIC_ID, 0.0), 20, BROKER_ENTITIES.get(0));
```

Comment:
Are we testing with `non-zero` variance in any of our tests?

Reply:
Add some variance in these test cases.